Makefile

```diff
@@ -240,8 +217,8 @@ infoblox-secret:
 
 .PHONY: license
 # updates source code with license headers
-license: golic
-	$(GOLIC) inject -c "2021 Absa Group Limited"
+license:
+	$(call golic,--dry -t apache2)
```

Comment:
Why is it dry?

Reply:
@somaritane , license target must modify files (injects license automatically if file is new). Sorry, don't know in which context we were talking about. Sorry if there was some misunderstandings caused by me.

Reply:
So `--dry` should be removed here, correct?

Reply:
> So `--dry` should be removed here, correct?

@ytsarev , yes should be removed

Reply:
👍  thanks for clarification @kuritka ! will amend.

Reply:
@kuritka @ytsarev amended